modules/datadict/php/datadict.class.inc

```diff
@@ -199,15 +138,14 @@ class Datadict extends \NDB_Menu_Filter
                 if ($val == 'empty') {
                     $query .= " AND COALESCE(pto.description,pt.description) = ''";
                 } elseif ($val=='modified') {
-                    //NO where clause added
+                    $query .= " AND pto.description IS NOT NULL";
```

Comment:
@xlecours isn't it possible that a field IS modified and the description IS null or is that not supposed to be done ?

i.e. a parameter description is "blablabla" and is was overridden with NULL ??

Reply:
when testing, if the description is null, it isnt considered modified, it's considered empty, but the empty filter isn't working as should (i assumed not part of this PR)

Reply:
the null value also gets overridden by the original description on the front-end but the change is logged in parameter_type_override table